sites/tv/site/layouts/_default/baseof.html

```diff
@@ -19,6 +19,11 @@
 
   {{ block "main" . }}{{end}}
 
+  {{ if not .IsHome }}
+    <!-- Footer in home page is directly added inside the hero -->
+    {{ block "footer" . }}{{ partial "footer" . }}{{end}}
+  {{ end }}
```

Comment:
The footer was missing on the 404 page for the tv site. Well, in fact, in all non-home pages. This fixes it.